cypress/e2e/test.cy.js

```diff
@@ -0,0 +1,49 @@
+//User loads the page//
+describe('On Page Load', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://api.thedogapi.com/v1/images/search?limit=100&api_key=live_00isfy9kzQCyFWWBludIQFj4g1pDwEoM87PH2PTVx8njhE7q1oEBDzg5lOhHq0QZ', {
+      statusCode: 200,
+      fixture: 'apiResponse.json', // This fixture should represent the initial state
+    }).as('apiRequest');
+    cy.visit('localhost:3000');
+    cy.wait('@apiRequest');
+  });
+
+
+  // // ON PAGE LOAD GRABBING API INFORMATION
+it('should GET all the dogs on page load', () => {
+  cy.get('.dogs-container').children().should('have.length', 4)
+  // expect the first() and last() cards content to be tested - testing the length of the cards alone is not enough
+  //  //These images represent the first and last images of the moveis api
+ cy.get('img[src="https://cdn2.thedogapi.com/images/IyR0sT3yy.jpg"]').should('be.visible');
+ cy.get('.dogs-container').first().should('contain', 'Akita');
+ cy.get('img[src="https://cdn2.thedogapi.com/images/Dtxn7vdC0.jpg"]').should('be.visible');
+ cy.get('.dogs-container').last().should('contain', 'Standard Schnauzer');
```

Comment:
@butlertree What you have here is good, but you could consider refactoring this a little bit, so that you're explicit about the image being part of the first (and last) card as well. 

You could do something like this:
```
cy.get('.dogs-container').within(() => {
  cy.get('img').first().should('have.attr', 'src').should('include', 'https://cdn2.thedogapi.com/images/IyR0sT3yy.jpg');
  cy.get('.dog-breed-name').first().should('contain', 'Akita');
  });
```

And then you could do the same for the last card in the container.